examples/cookies/index.js

```diff
@@ -19,7 +19,7 @@ if (process.env.NODE_ENV !== 'test') app.use(logger(':method :url'))
 app.use(cookieParser('my secret here'));
 
 // parses x-www-form-urlencoded
-app.use(express.urlencoded())
+app.use(express.urlencoded()({ extended: false }))
```

Comment:
The proper way of setting options is

```suggestion
app.use(express.urlencoded({ extended: false })
```